kolibri/plugins/management/assets/src/device_management/views/manage-content-page/channels-grid.vue

```diff
@@ -60,8 +60,7 @@
 <script>
 
```

Comment:
Add an `out-in` transition around the v-ifs in the template to avoid this

![jump](https://user-images.githubusercontent.com/7193975/30128893-1371d076-92f9-11e7-811f-02d41eadffad.gif)

https://vuejs.org/v2/guide/transitions.html#Transition-Modes